packages/world/src/modules/core/CoreModule.sol

```diff
@@ -117,7 +117,9 @@ contract CoreModule is IModule, WorldContext {
       WorldRegistrationSystem.registerSystemHook.selector,
       WorldRegistrationSystem.registerSystem.selector,
       WorldRegistrationSystem.registerFunctionSelector.selector,
+      WorldRegistrationSystem.registerFunctionSelectors.selector,
       WorldRegistrationSystem.registerRootFunctionSelector.selector,
+      WorldRegistrationSystem.registerRootFunctionSelectors.selector,
```

Comment:
@dk1a @alvrs What do you think of just using the plural here and not the singular?

Reply:
this isn't a hot path so I think it's fine to ditch the singular, a little extra gas during deployment shouldn't be a big deal